geo/Geoserver.py

```diff
@@ -492,6 +492,9 @@ def create_layer_rule(self, workspace: str, layer: str, permission: str, role: s
     def update_layer_rule(self, workspace: str, layer: str, permission: str, role: str):
         """
         Update a current security rule for the specified layer(s) within a workspace
+        TODO: Currently, it is not possible to add more than one role to a rule within the same 
+        workspace. The expected behavior is to have the option to not only replace the rule but also
```

Comment:
This is also wrong, the expected behavior is to be able to replace, add or remove the role, but the rule would be kept as it is

Reply:
Done!